docs/monitoring/resources.rst

```diff
@@ -0,0 +1,46 @@
+.. |MonitorDiskUsge| image:: ../images/MonitorDiskUsge.jpg
+
+Resources
+=========
+
+Monitoring resource usage within a Fledge instance can be achieved by using the south plugin that is designed to monitor the Linux resources of the machine on which Fledge is running, this is the systeminfo south plugin.
+
+This plugin creates a number of assets with a configurable prefix on the asset name. This include
+
+  - The aggregate CPU usage for all the CPUs within the machine.
+
+  - The disk traffic for that disk for each disk connected to the machine.
+
+  - The load average of the machine.
+
+  - The memory usage information for the machine.
+
+  - The network usage for each network interface on the machine.
+
+  - The paging and swapping events for the machine.
+
+  - Platform information for the machine.
+
+  - A summary of the processes running on the machine.
+
+There are some other things that are also monitored that of not of particular interest for monitoring resource usage.
+
+Example
+-------
+
+In this example we will assume we want to use the notification service to monitor the percentage of space remaining on a particular disk within the Fledge machine. We use the systeminfo plugin to get the data we want, in this case we will look at the disk device sda1, using the defaults of the plugin this will create an asset called *system/diskUsage_dev/sda1* with a datapoint called *Use_prcntg*.
```

Comment:
- Start new sentence at "In this case we will look at the disk."
- Start new sentence at "Using the defaults of the plugin."